packages/jest-expect-har/package.json

```diff
@@ -4,8 +4,17 @@
   "description": "Jest/Vitest matcher for asserting valid HAR definitions",
   "license": "MIT",
   "author": "Jon Ursenbach <jon@readme.io>",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
```

Comment:
I don't _think_ that this is a breaking release since these all should be backwards compatible but happy to tag another major on this library if you think its worth it.

Reply:
if it’s a single default export then yeah probably not breaking

Reply:
confirmed `node10` still works so im not going to treat this as a breaking release